axelrod/strategies/memoryone.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, ep=0.05):
         self.set_four_vector(four_vector)
 
 
-class ZeroDeterminantPlayer(MemoryOnePlayer):
+class LRPlayer(MemoryOnePlayer):
```

Comment:
That's what they are called in the reference. It stands for Linear Relation -- do you prefer that?